Stack/Opc.Ua.Core/Stack/Tcp/TcpServerChannel.cs

```diff
@@ -953,6 +959,26 @@ private bool ProcessRequestMessage(uint messageType, ArraySegment<byte> messageC
                 }
             }
         }
+
+        /// <summary>
+        /// Checks if the ChunkOrSizeLimitsExceeded is set, raises a servicefault and resets the flag
+        /// </summary>
+        protected override void ResetMessageLimitsExceeded(ChannelToken token, uint requestId)
+        {
+            if (ChunkOrSizeLimitsExceeded)
+            {
+                const string message = "MaxChunkCount or MaxMessageSize exceeded";
```

Comment:
Probably a simple and more blunt alternative would be to call ChannelClosed() and interrupt the communication with the client entirely